packages/legend-query/src/stores/QueryBuilderSetupState.ts

```diff
@@ -35,6 +36,7 @@ import {
 export class QueryBuilderSetupState {
   queryBuilderState: QueryBuilderState;
   _class?: Class | undefined;
+  versionPropertyParameter?: ValueSpecification | undefined;
```

Comment:
Can you file another issue. We should eventually give users the right to edit this. This will require more design discussion. Example if they want to change the from `$businessDate` to `$lastest` or an instance value, the UI should support that.